src/sidebar/components/top-bar.js

```diff
@@ -40,8 +40,17 @@ function TopBar({
 
   const pendingUpdateCount = useStore(store => store.pendingUpdateCount());
 
+  const togglePanelFn = useStore(store => store.toggleSidebarPanel);
+  const currentActivePanel = useStore(
+    store => store.getState().sidebarPanels.activePanelName
+  );
+
   const applyPendingUpdates = () => streamer.applyPendingUpdates();
 
+  const toggleSharePanel = () => {
```

Comment:
Do you know if we tracking work to update UI here to match this new standard? e.g. Show the toggled state by making icon darker.

Reply:
The other fully-toggle-able thing in the top-bar is the help panel. I've got that work (https://github.com/hypothesis/client/issues/1203) teed up for next sprint.

I have a little work to do to organize some follow-on issues/tasks regarding broader conversations around UI states, focus, a11y, etc., across these components. On my list!